tests/elements-regression/src/global-setup.js

```diff
@@ -0,0 +1,123 @@
+const { chromium, request } = require( '@playwright/test' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const WpAdminPage = require( './pages/wp-admin-page.js' );
+
+module.exports = async ( { projects: [ { use: config } ] } ) => {
+	const { page, browser } = await createPage( {
+		headless: config.headless,
+		baseURL: config.baseURL,
+	} );
+
+	const { storageStateObject, wpRESTNonce } = await login( page, {
+		username: config.user.username,
+		password: config.user.password,
+		storageState: config.storageState,
+	} );
+
+	const apiContext = await createApiContext( {
+		storageStateObject, wpRESTNonce, baseURL: config.baseURL,
```

Comment:
```suggestion
		storageStateObject, 
		wpRESTNonce, 
		baseURL: config.baseURL,
```

Just to be consistent with the other objects

Reply:
Done